### PR TITLE
Upgrade image cropping library to latest version (New PR)

### LINKED ISSUE #4454: Crash report 5.6: IAE in com.yalantis.ucrop.task.BitmapCropTask.crop
It comes from the `com.yalantis.ucrop` library, but it's one of the top crash, we should fix the lib.

```
Fatal Exception: java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by java.lang.IllegalArgumentException: width must be > 0
       at android.graphics.Bitmap.checkWidthHeight(Bitmap.java:373)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:664)
       at android.graphics.Bitmap.createBitmap(Bitmap.java:633)
       at com.yalantis.ucrop.task.BitmapCropTask.crop(BitmapCropTask.java:144)
       at com.yalantis.ucrop.task.BitmapCropTask.doInBackground(BitmapCropTask.java:88)
       at com.yalantis.ucrop.task.BitmapCropTask.doInBackground(BitmapCropTask.java:24)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

### PR DESCRIPTION
**Pull request by hypest**
Fixes #4454

To test:

No specific steps to test as there are no steps to reproduce the original issue. Instead, try to change your Gravatar through the app by cropping some picture. It should work.

### CODE REVIEW

**Comment by nbradbury** [comment]
:shipit:

File: WordPress/build.gradle [hunk 1]
```diff
@@ -105,13 +105,14 @@ dependencies {
...
 
-    compile 'com.yalantis:ucrop:1.5.0'
+    compile 'com.yalantis:ucrop:2.2.0'
     compile 'com.github.xizzhu:simple-tool-tip:0.5.0'
 
     androidTestCompile 'com.google.dexmaker:dexmaker-mockito:1.0'
     androidTestCompile 'org.objenesis:objenesis:2.1'
     androidTestCompile 'org.mockito:mockito-core:+'
     androidTestCompile 'com.squareup.okhttp:mockwebserver:2.7.5'
+    androidTestCompile 'com.squareup.okio:okio:1.9.0' // explicitly compile okio to match the version needed by ucrop
```
Comment (maxme) [inline WordPress/build.gradle]: Not sure to understand why it's an `androidTestCompile` entry?

Comment (hypest) [inline WordPress/build.gradle]: `compile 'com.yalantis:ucrop:2.2.0'` pulls in v1.9.0 at the time of writing, while tests are pulling in v1.6.0. I explicitly added this to align the versions and stop gradle from complaining. Should I use some other, better method of aligning those?

Comment (maxme) [inline WordPress/build.gradle]: Ah I see, `com.squareup.okhttp:mockwebserver` is dependent on okio v1.6, looks good.

**Pull request merged**